apps/fluent-tester/src/FluentTester/FluentTester.android.tsx

```diff
@@ -0,0 +1,115 @@
+
+export interface FluentTesterProps {
+  initialTest?: string;
+  enabledTests: TestDescription[];
+}
+
+const Header: React.FunctionComponent<{}> = () => {
+  const theme = useTheme();
+
+  return (
+    <View style={{ ...fluentTesterStyles.header, flexDirection: 'column' }}>
+      <Text style={[fluentTesterStyles.testHeader]} variant="heroSemibold" color={theme.host.palette?.TextEmphasis} testID={BASE_TESTPAGE}>
+        ⚛ FluentUI Tests
+      </Text>
+      <ScrollView horizontal={true} showsHorizontalScrollIndicator={false}>
+        <ThemePickers />
+      </ScrollView>
+    </View>
+  );
+};
+
+const getThemedStyles = themedStyleSheet((t: Theme) => {
+  return {
+    root: {
+      backgroundColor: t.colors.background,
+      flexDirection: 'column',
+      height: '100%',
+      justifyContent: 'flex-start',
+      alignItems: 'stretch',
+      padding: 4,
+    },
+  };
+});
+
+export const FluentTester: React.FunctionComponent<FluentTesterProps> = (props: FluentTesterProps) => {
+  // sort tests alphabetically by name
+  const sortedTestComponents = props.enabledTests.sort((a, b) => a.name.localeCompare(b.name));
+
+  const [selectedTestIndex, setSelectedTestIndex] = React.useState(0);
+  const [onMainScreen, setOnMainScreen] = React.useState(true);
+
+  const TestComponent = sortedTestComponents[selectedTestIndex].component;
+
+  const themedStyles = getThemedStyles(useTheme());
+
+  const onBackPress = () => {
+    setOnMainScreen(true);
+    BackHandler.removeEventListener('hardwareBackPress', onBackPress);
+    return true;
+  };
+
+  return (
+    <DisplayIfVisible isVisible={true}>
+      <DisplayIfVisible isVisible={onMainScreen}>
+        <View style={themedStyles.root}>
+          <Header />
+          <Separator />
+          <View style={{ ...fluentTesterStyles.testRoot, marginTop: 10 }}>
+            <ScrollView style={fluentTesterStyles.testList} contentContainerStyle={fluentTesterStyles.testListContainerStyle}>
+              {sortedTestComponents.map((description, index) => {
+                return (
+                  <View key={index}>
+                    <Text
+                      key={index}
+                      onPress={() => {
+                        setOnMainScreen(false);
+                        setSelectedTestIndex(index);
+                        BackHandler.addEventListener('hardwareBackPress', onBackPress);
+                      }}
+                      style={mobileStyles.testListItems}
+                      testID={description.testPage}
+                    >
+                      {description.name}
+                    </Text>
+                    <Separator style={mobileStyles.testListSeparator} />
+                  </View>
+                );
+              })}
+            </ScrollView>
+          </View>
+        </View>
+      </DisplayIfVisible>
+      <DisplayIfVisible isVisible={!onMainScreen}>
```

Comment:
Oh I see.. you're making "pages" by having different parts render based on the boolean. 
"Main Screen" sounds like screen 1 if we had a multi display configuration (like an external monitor). Could we name this something like "onTestListView" or something more specific?

Or maybe just split it out to a separate component with comments to describe the pattern?

Reply:
Maybe too complicated, but rather than have a completely android specific page, maybe we could add the "DisplayIfVisible" wrapper to FluentTester.tsx, and use a "useTwoPanes" flag that is set appropriately for each platform, then we can share the logic? And with this review, have it default to only use the new one pane solution for android, which is what you've tested.

That way, we could have useTwoPanes set to `true` on Desktop and Tablet (Or Surface Duo.. however that works) and we get the original tester, and `false` on iOS and android, and we get your new tester logic.

Reply:
Thanks! I like this approach =)